style.css

```diff
@@ -259,6 +250,29 @@ body {
     margin-left: 30px;
   }
 
+  .popup-skills > li {
+    font-family: 'Poppins', 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #3c3a39;
+    border: 1px solid #dfe1e6;
+    margin-left: 10px;
+  }
+
+  .skillset > li {
+    font-family: Inter, sans-serif;
+    margin-right: 12px;
+    padding: 6px;
+    border: 1px solid #8993a4;
+    font-style: normal;
+    font-weight: 500;
```

Comment:
- [ ] I would urge you to work on all the changes i requested for previously, I notice you didn't implement some of the changes require. 

- On the desktop view kindly stick to the following requirement.
    - Kindly remove the `<br>` tag you gave the header text so that it can be on a straight line and not broken as seen in the Figma design
    - Secondly the space between the image and the text is much kindly reduce the padding-left to 1rem. On the class `.side-text`
    - Also the buttons as seen in the Figma design align with the image in the bottom, i would suggest you make the margin-top in the class `.demoButtons` to be `43%`,
    - On the class `.popup-text` kindly give it a `width: 90%;`
    - Kindly give a blur background as we have it in the Figma design

- On the mobile view kindly stick to the following requirement.
    -  On id `#details-popup`, modify the property to be `width: 90%;` and `margin: 24px 5%;`
    - I would also advice you work on making the background becomes blur once the card is popped as seen in the Figma design

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/78634841/233383886-bc57e432-53a7-4459-af5b-a890446c2590.png) | ![real](https://user-images.githubusercontent.com/78634841/233339134-13bce593-8db3-4c0a-9afb-129eb6645bd3.png)  |


- [ ] I would kindly suggest you create a pop up modal for the first cards as well in your recent work section of your page to make it 7 cards in total. Currently, i notice the pop up functionality is not implemented for the first card. Kindly work on it. Thank you 👍 👍

Reply:
Hi,

All the changes regarding property values have been incorporated . 

However, I would like to clarify that the number of cards in the Figma design was not explicitly stated, so I made an assumption based on what I saw. I understand that you have a different assumption, and I respect your perspective. I would like to kindly request that we stick to the design that I have presented, as I believe it best represents what was there in the Figma file and to maintain consistency as well from the previous approved projects.

Regarding the use of <br> tags in the design, I apologize if this caused any confusion. I placed them intentionally to accommodate the text and other elements in the design. As we were instructed to replicate the design as closely as possible, I took some artistic liberty to ensure that the design looked accurate and visually pleasing. I have put significant effort into creating the current design replicating as closely as possible based on the original instructions on Figma design.

Thanks.